gui-wm/sway/sway-9999.ebuild

```diff
@@ -14,13 +14,13 @@ if [[ ${PV} == 9999 ]]; then
 else
 	MY_PV=${PV/_rc/-rc}
 	SRC_URI="https://github.com/swaywm/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
```

Comment:
keywords should remain the same unless there's a reason for removal (which should probably be it's own review)

Reply:
oh, old because it's an old request, sorry for not noticing

Reply:
Hi @prometheanfire 
No problem, i've rebased my PR and reverted the dropped `KEYWORDS` for the live ebuild.